java/TwentyOne.java

```diff
@@ -0,0 +1,15 @@
+public class TwentyOne {
+    public static void main(String[] args) {
+        int a = 10;
+        int b = 17;
+        int c = 9;
+
+        if (a > b & b > c) {
```

Comment:
I detect that this code is problematic. According to the [Correctness (CORRECTNESS)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#correctness-correctness), [SA: Nonsensical self computation involving a value (e.g., x & x) (SA_SELF_COMPUTATION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#sa-nonsensical-self-computation-involving-a-field-e-g-x-x-sa-field-self-computation).
This method performs a nonsensical computation of a field with another reference to the same field (e.g., x&x or x-x). Because of the nature of the computation, this operation doesn't seem to make sense, and may indicate a typo or a logic error. Double check the computation.